### PR TITLE
Patch vulnerabilities test

### PR DESCRIPTION
**Pull request by gianluca-isgro**
as agreed I send you pull request

### CODE REVIEW

File: src/main/java/it/eng/idsa/businesslogic/configuration/OkHttpClientConfiguration.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import java.security.KeyManagementException;
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertificateException;
+import java.util.Collections;
```
[the remark this answers is not in the document]
Comment (gianluca-isgro) [reply src/main/java/it/eng/idsa/businesslogic/configuration/OkHttpClientConfiguration.java:6]: import java.util.Collections;
import okhttp3.ConnectionSpec;

These?

After  I delete the import do I have to send a new pull request?

File: ci/docker/ecc_resources_consumer/logback-SENDER.xml [hunk 2]
```diff
@@ -9,8 +9,9 @@
     </encoder>
   </appender>
 
+	<property name="LOG_DIR" value="/var/log/ecc" />
```
[the remark this answers is not in the document]
Comment (gianluca-isgro) [reply ci/docker/ecc_resources_consumer/logback-SENDER.xml:12]: In new Dockerfile I create this path for the ecc log.

I would do the same thing for be_data_app but there aren't logback configuration files or I can't find them.

[the remark this answers is not in the document]
Comment (IgorBalog-Eng) [reply ci/docker/ecc_resources_consumer/logback-SENDER.xml:12]: For dataApp, we still do not have logback, this is something we could add, in dataApp and in UsageControl dataApp, to be compatible

**Comment by IgorBalog-Eng** [comment]
No critical or high security vulnerabilities are present with new docker image, and library update.
From Base Image                    Vulnerabilities  Severity
openjdk:11.0.15-jre-bullseye  92               8 critical, 10 high, 8 medium, 66 low to  
Base Image    Vulnerabilities  Severity
ubuntu:22.04  11               0 critical, 0 high, 2 medium, 9 low

**Review by IgorBalog-Eng — APPROVED** [review]

**Pull request merged**